plans/libevent/plan.sh

```diff
@@ -0,0 +1,13 @@
+pkg_origin=core
+pkg_name=libevent
+pkg_version=2.0.22
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
```

Comment:
Ah cool - wasn't sure how this was supposed to work (if it was the Habitat license, the license I'd want my plan.sh under, or the upstream source code license) - thanks for clarifying.